src/core/client.ts

```diff
@@ -845,7 +936,7 @@ export class Client<P = Indexable> implements Observable<ClientEvent<P>> {
         attachment.remoteChangeEventReceived = true;
         this.eventStreamObserver.next({
           type: ClientEventType.DocumentsChanged,
-          value: respDocKeys,
+          value: [docKey],
```

Comment:
Would it be better to change the DocumentsChanged event to singular(DocumentChanged)?

Reply:
Yes, as you suggested, I think it would be better to change. 
I'm planning to create a separate issue for that.